src/main/java/org/example/game/War.java

```diff
@@ -193,47 +202,25 @@ private void printCardsInPlay() {
         }
     }
 
-
     @Override
     public boolean playAgain() {
+
         System.out.println("Do you want to play again? (yes/no): ");
         Scanner scanner = new Scanner(System.in);
         String playAgain = scanner.nextLine();
-        if (playAgain.equalsIgnoreCase("yes")) {
-            return true; // Return true if the user wants to play again
-        } else {
-            System.out.println("Thanks for playing!");
-            return false; // Return false if the user doesn't want to play again
-        }
-    }
-
-    //for testing
-    // Getters and setters for player hands
-    public UserInteraction getBill() {
-        return bill;
-    }
-
-    public UserInteraction getPlayerOne() {
-        return playerOne;
+        return playAgain.equalsIgnoreCase("yes"); // Return true if the user wants to play again
```

Comment:
This is how the playAgain method should look 👍 it's up to you whether to use input validation or not but it's included in the Commands class in case you'd prefer to use that, though the uuser can only input numbers

Reply:
cool will add it in thank you :-)